src/lib.rs

```diff
@@ -117,9 +117,13 @@ pub use peer_handle::PeerHandle;
 pub use peer_handle::PeerCloseHandle;
 pub use peer_handle::PeerReadHandle;
 pub use peer_handle::PeerWriteHandle;
-pub use request::ReceivedMessage;
-pub use request::ReceivedRequest;
-pub use request::SentRequest;
+pub use request::{
+	ReceivedMessage,
+	ReceivedRequestHandle,
+	ReceivedRequestWriteHandle,
+	SentRequestHandle,
+	SentRequestWriteHandle,
+};
```

Comment:
Can we not follow this for peer_handle and server also? It looks better in my opinion :P

Reply:
Yeah, but lets do that outside of this PR. It can probably be automated with rustfmt :)